jmclient/jmclient/blockchaininterface.py

```diff
@@ -829,10 +829,30 @@ def pushtx(self, txhex):
         return True
 
     def query_utxo_set(self, txout, includeconf=False, includeunconf=False):
+        """If txout is either (a) a single string in hex encoded txid:n form,
+        or a list of the same, returns, as a list for each txout item,
+        the result of gettxout from the bitcoind rpc for those utxs;
+        if any utxo is invalid, None is returned.
+        includeconf: if this is True, the current number of confirmations
+        of the prescribed utxo is included in the returned result dict.
+        includeunconf: if True, utxos which currently have zero confirmations
+        are included in the result set.
+        If the utxo is of a non-standard type such that there is no address,
+        the address field in the dict is None.
+        """
         if not isinstance(txout, list):
             txout = [txout]
         result = []
         for txo in txout:
+            if len(txo) < 66:
+                result.append(None)
+                continue
+            try:
+                txo_idx = int(txo[65:])
```

Comment:
`txo_idx` should be used below I guess?

Reply:
Yeah doesn't matter, but I agree feels slightly cleaner that way.